packages/key-did-resolver/src/index.ts

```diff
@@ -0,0 +1,29 @@
+import type { ParsedDID, DIDResolver, DIDDocument } from 'did-resolver'
+
+interface ResolverRegistry {
+    [index: string]: DIDResolver;
+}
+
+import * as secp256k1 from './secp256k1/secp256k1-driver'
+
+// supported drivers
+const prefixToDriverMap: any = {
+    zQ3s: secp256k1,
+}
+
+export default {
+    getResolver: (): ResolverRegistry => ({
+        'key': async (did: string, parsed: ParsedDID): Promise<DIDDocument | null> => {
+            if (did.indexOf('did:key:') !== 0) {
+                throw new Error('did must be of method did:key.');
+            }
+            const idchar: any = did.split('did:key:').pop();
```

Comment:
we can just use id from the parsedDID: https://github.com/decentralized-identity/did-resolver/blob/1631fa91cac0888c039c84a543855193081273e3/src/resolver.ts#L63